package.json

```diff
@@ -47,12 +49,21 @@
     "packages/tentales-router",
     "packages/tentales-router-plugin-https-only"
   ],
+  "jest": {
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
```

Comment:
I really feel like the tests should be colocated with whatever they're testing.

Reply:
Tru.